### DISCUSSION TITLE
How to use Kornia to process tiff images and make them clearer

### DISCUSSION THREAD

**Discussion by monkeycc** [kornia/kornia#2572] [Questions]
![未标题-1](https://github.com/kornia/kornia/assets/6490927/812a38af-12a9-43e3-a2d0-bacdc5f09e3b)

Download tiff image file:

[1.zip](https://github.com/kornia/kornia/files/12727256/1.zip)

**Comment by edgarriba** [comment]
you can try different equalization techniques: https://kornia.readthedocs.io/en/latest/enhance.html#equalization
there is a tutorial here too: https://kornia.github.io/tutorials/nbs/image_histogram.html

**Comment by monkeycc** [comment]
Unfortunately
Unable to adjust


It seems that it needs to be converted to HDR to adjust the details
But Kornia does not yet support converting HDR

**Reply by edgarriba** [reply]
@monkeycc can you suggest or contribute with a HDR conversion algorithm ?

**Comment by monkeycc** [comment]
Gimp
Sharpen (Unsharp Mask)
https://github.com/GNOME/gimp


Can be achieved
Gimp is open source
But I don't know how to look at the algorithm
@edgarriba

**Reply by edgarriba** [reply]
in case of unsharp -> https://kornia.readthedocs.io/en/latest/filters.html#kornia.filters.unsharp_mask

**Reply by monkeycc** [reply]
```python
sharpen = kornia.filters.UnsharpMask((9, 9), (10, 10))
sharpened_tensor = sharpen(data)
sharpened_image = kornia.utils.tensor_to_image(sharpened_tensor)
```

Unfortunately
Adjusted some parameters
Unable to perform clarity
